lms/templates/navigation.html

```diff
@@ -112,52 +162,66 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         </ol>
       % endif
     % else:
-    <ol class="left nav-global">
+    <ol class="left ${main_nav_menu_classlist}">
       <%block name="navigation_global_links">
         % if static.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)):
-          <li class="nav-global-01">
+          <li class="item nav-global-01">
             <a href="${marketing_link('HOW_IT_WORKS')}">${_("How it Works")}</a>
           </li>
           % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
-            <li class="nav-global-02">
+            <li class="item nav-global-02">
               <a href="${marketing_link('COURSES')}">${_("Courses")}</a>
             </li>
           % endif
-          <li class="nav-global-03">
+          <li class="item nav-global-03">
             <a href="${marketing_link('SCHOOLS')}">${_("Schools")}</a>
           </li>
         % endif
       </%block>
 
       <%block name="navigation_other_global_links">
+      <%
+      if uses_pattern_library:
+          discovery_btn_classlist = "btn"
+          register_btn_classlist = "btn-neutral"
+      else:
+          discovery_btn_classlist = "cta cta-discovery"
+          register_btn_classlist = "cta cta-register"
+      %>
```

Comment:
Would it be possible to move all of these `if uses_patter_library` checks to a central location?